connectors/v2/playmoss.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+/* global Connector */
+
+Connector.playerSelector = '#footer-player';
+
+Connector.artistTrackSelector = '#footer-player li.playing a';
+
+Connector.playButtonSelector = 'button.play';
+
+Connector.getArtist = function() {
+	var text = $(this.artistTrackSelector).text().split(' - ')[0].trim();
+	return text || null;
+};
+
+Connector.getTrack = function() {
```

Comment:
This is no longer required either since the work is done in the BaseConnector via the artistTrackSelector property